config/database.yml

```diff
@@ -1,25 +1,18 @@
-# SQLite. Versions 3.8.0 and up are supported.
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
 default: &default
   adapter: postgresql
+  encoding: unicode
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
 
 development:
   <<: *default
-  database: db/development
+  database: jurgita_art_development
 
-# Warning: The database defined as "test" will be erased and
-# re-generated from your development database when you run "rake".
-# Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: db/test
+  database: jurgita_art_test
 
 production:
   <<: *default
-  database: db/production
+  database: jurgita_art_production
```

Comment:
why did you rename the dbs?

Reply:
Because I changed them to postgres DB names from sqlite db paths.